sw/device/lib/dif/dif_i2c.h

```diff
@@ -90,6 +90,24 @@ typedef struct dif_i2c_timing_config {
   uint32_t scl_period_nanos;
 } dif_i2c_timing_config_t;
 
+/**
+ * Configuration for the addressing behavior of the I2C, can be disabled or
+ * configured to look for multiple addresses by masking certian bits.
+ */
+typedef struct dif_i2c_id {
+  /**
+   * Mask the I2C transactions before checking for a match. Recieved Address &
```

Comment:
Nits: Maybe something like...
```suggestion
   * Mask the I2C address before checking for a match. Received Address &
```

Reply:
I'll try to keep from mispelling recieved and you're right it's clearer here if this says address, but I'll also specify it's the incoming address to make it clear it's not the specified address that gets masked.